scipy/optimize/_differentialevolution.py

```diff
@@ -132,6 +135,18 @@ def differential_evolution(func, bounds, args=(), strategy='best1bin',
         ``np.std(pop) <= atol + tol * np.abs(np.mean(population_energies))``,
         where and `atol` and `tol` are the absolute and relative tolerance
         respectively.
+    aggressive : bool, optional
+        The best solution vector is continuously updated within a single
+        generation. This can lead to faster convergence as trial vectors can
+        take advantage of continuous improvements in the best solution.
+    workers : int or Pool-like object, optional
+        If `workers` is an int the population is subdivided into `workers`
+        sections and evaluated in parallel (using `multiprocessing.Pool`).
+        Supply `-1` to use all cores available to the Process.
+        Alternatively, supply an object with a map method, such as
+        `multiprocessing.Pool`, for evaluating the population in a parallel
+        fashion. This option is only applicable if `aggressive` is set to
+        False.
```

Comment:
> This option is only applicable if `aggressive` is set to False.

What about having workers override aggressive? That way a user will only have to specify workers, and not both.

I was running this code and wondering why workers wasn't doing anything at first.

The change in the docstring would looks something like
>This option automatically sets `aggressive` to False

Reply:
I'll have a think about the API for this. However it needs to be explicit, which won't be the case if workers has the effect of setting aggressive as well.